snapcraft/plugins/jhbuild.py

```diff
@@ -284,11 +284,10 @@ def _setup_jhbuild(self):
 
         if jhbuild_user != os.getuid():
             chmod_path = os.path.dirname(os.path.dirname(self.partdir))
-
-        for filename in glob.iglob('%s/**' % chmod_path, recursive=True):
-            if not os.path.exists(filename):
-                continue
-            os.chown(filename, jhbuild_user, jhbuild_group)
+            for filename in glob.iglob('%s/**' % chmod_path, recursive=True):
```

Comment:
now that we are here, could you please use `.format` instead of `%`?

Reply:
done

Reply:
actually, it's not drop-in. I screwed that up.

Reply:
that's better.